composer.json

```diff
@@ -6,7 +6,7 @@
     "license": "MIT",
     "keywords": ["october", "debugbar"],
     "require": {
-        "barryvdh/laravel-debugbar": "~3.0"
+        "barryvdh/laravel-debugbar": "~3.10.3"
```

Comment:
This is good but it does stick it in the 3.10.x range. Which is safer, but might not be intentional (vs ^3.10.3)